### DISCUSSION TITLE
Problem with widget on macOS

### DISCUSSION THREAD

**Discussion by Czerpii** [TomSchimansky/CustomTkinter#1770] [Q&A]
Hi, i have a problem with customtkinter on macOS. I paste to VSCode example code from GitHub. When I tried open it everything looks fine. But I can't see any widget. 
`import customtkinter

app = customtkinter.CTk()  # create CTk window like you do with the Tk window
app.geometry("400x240")

def button_function():
    print("button pressed")

button = customtkinter.CTkButton(master=app, text="CTkButton", command=button_function)
button.pack()

app.mainloop()`

There is a code I use. 
I can press the button and I get message "button pressed" but I can't see it.
Any suggestion?

**Comment by avalon60** [comment]
The code works fine. Where are you expecting to see the message? 
The message won't appear in the GUI, if that is what you expect.

![image](https://github.com/TomSchimansky/CustomTkinter/assets/89534395/892fdd4d-4a76-464a-ae31-7e63834e2fa7)

**Comment by Czerpii** [comment]
I got it. I need to reinstall python on my Mac. Idk why...

**Comment by Akascape** [comment]
@Czerpii 
Duplicate of https://github.com/TomSchimansky/CustomTkinter/issues/1690

In short, mac os tkinter is buggy, only solution is to update python.
